baserepo.py

```diff
@@ -53,52 +56,53 @@ def from_url(cls, url, token):
         match = cls.PATH_RE.match(parts.path)
         if not match:
             raise RepoError(
-                "Bad path. Can't separate namespace from "
-                "repo name {}".format(parts.path)
+                "Bad path. Can't separate namespace " +
+                "from repo name {}.".format(parts.path)
             )
 
         namespace = match.group("namespace")
         name = match.group("name")
 
-        self = cls(urlunsplit((parts.scheme, parts.netloc, '', '', '')), namespace, name, token, url)
+        self = cls(urlunsplit((parts.scheme, parts.netloc, "", "", "")),
+                   namespace, name, token, url)
 
         logging.debug(json.dumps(self.info))
-        logging.info("Found %s", self.name_with_namespace)
+        logging.info("{} is valid.".format(self.name_with_namespace))
```

Comment:
Should this be a debug-level log? It is weird seeing it in the output all the time.